### ISSUE TITLE
[Idea] new minigame

### ISSUE THREAD

**Issue by swarfey** [fokkonaut/F-DDrace#59]
new minigame like at the beginning of [this video](https://www.youtube.com/watch?v=ofGWeGkQ_UQ) with a bot like MLG_PRO on ChillBlock5 (and with a real player, if anyone is there)
![image](https://user-images.githubusercontent.com/45124381/101266029-d923a980-374b-11eb-9806-a12f3b0a0dc5.png)

**Comment by ChillerDragon** [comment]
Such a weird minigame :D reminds me of ddnet++ balance battle minigame. Can't you just do that move anywhere in the map with your friends or dummy? Maybe the police bot could be extended to do this move in the police area so you can train it there.

**Comment by swarfey** [comment]
> Can't you just do that move anywhere in the map with your friends or dummy? 
yes, you can, but it'd be cooler to have an extra minigame for that, which would have an extra /top5, rewards for each 10s or so alive, etc..
